src/Traits/ExceptionsTestTrait.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace RonasIT\Support\Traits;
+
+trait ExceptionsTestTrait
+{
```

Comment:
let's aggregate all testing traits under the one

```suggestion
trait TestingTrait
{
    use MailsMockTrait;
    use FixturesTrait;
    use MockTrait;
```